crates/ide-db/src/path_transform.rs

```diff
@@ -9,6 +9,14 @@ use syntax::{
     ted, SyntaxNode,
 };
 
+#[derive(Default)]
+struct Substs {
```

Comment:
nit, let's rename this to `AstSubsts` as we might have `Substs` in the hir layer in the future